page-tracking/src/hw_mem_map.rs

```diff
@@ -88,9 +88,6 @@ pub enum HwReservedMemType {
     /// The hypervisor per-CPU memory area.
     HypervisorPerCpu,
 
-    /// Sv48 page tables for the hyeprvisor running in HS mode.
-    HypervisorPtes,
-
```

Comment:
Removal of `HypervisorPtes` should probably be in its own commit.

Reply:
Ack.

Reply:
Done.